appUITests/Screens/Accounts/AccountsListScreen.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AccountsListScreen.swift
+//  GoldenSampleUITests
+//
+//  Created by Backbase on 15/12/2023.
+//
+
+import Foundation
+import XCTest
+
+class AccountsListScreen : BaseScreen {
+
+    //MARK: ELEMENTS
+    private lazy var myAccountsHeaderLbl = app.staticTexts["My Accounts"]
+    private lazy var searchAccountTf = app.searchFields.firstMatch
```

Comment:
```suggestion
    private lazy var searchAccountSf = app.searchFields.firstMatch
```
I assume `Tf` should be replaced to `Sf` - search field?